test-utils/build.gradle

```diff
@@ -8,7 +8,7 @@ dependencies {
     implementation project(":common")
     implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion"
     latestDependenciesImplementation "com.android.billingclient:billing:$billingVersion"
-    unityIAPCompileOnly "com.android.billingclient:billing:$billingClient3Version"
```

Comment:
Do we need to remove the `billingClient3Version` property in the root `build.gradle`?